h5p-bildetema/src/components/Bildetema/Bildetema.tsx

```diff
@@ -13,19 +14,32 @@ import { TopicRouteController } from "../TopicRouteController/TopicRouteControll
 import { sanitizeLanguages } from "../../utils/language.utils";
 import styles from "./Bildetema.module.scss";
 import SearchPage from "../SearchPage/SearchPage";
+import CustomViewPage from "../CustomViewPage/CustomViewPage";
+import {
+  useCurrentLanguage,
+  useCurrentLanguageCode,
+} from "../../hooks/useCurrentLanguage";
 
 type BildetemaProps = {
   defaultLanguages: string[];
   isLoadingData: boolean;
 };
 
-const staticPaths = ["/sok"];
+const STATIC_PATHS = {
+  SEARCH: "/sok",
+  CUSTOM_VIEW: "/customview",
```

Comment:
If we use "customview" (for simplification?) maybe "sok" should change to "search"?

Reply:
agree, can fix this in a later pr